Sources/StevensonCore/SlackService.swift

```diff
@@ -7,38 +7,18 @@ public typealias SlackCommand = (
     parse: (SlackCommandMetadata) throws -> Command
 )
 
-public struct SlackCommandMetadata: Decodable {
+public struct SlackCommandMetadata: Content {
     public let token: String
-    public let channelName: String
+    public let channel_name: String
```

Comment:
Why going with snake case?

Reply:
There is no key decoding strategy in `URLEncodedFormDecoder` (data from slack is passed as url encoded string, not as json), and I didn't want to define all the keys 🤷‍♂️

Reply:
👍